veps/sig-network/ns-admin-nichotplug.md

```diff
@@ -0,0 +1,225 @@
+depends on a user-initiated migration following each NIC hotplug / hotunplug operation.
+The migration is required in order to attach / detach secondary interfaces to / from the target virt-launcher pod.
+
+> [!NOTE]
+> KubeVirt also supports in-place hotplug / hotunplug operations (without performing a migration)
+> by integrating with the [dynamic-networks-controller](https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller).
+>
+> This method does not support hot-plugging SR-IOV NICs, as SR-IOV hotplug operation requires a migration in order for the
+> [SR-IOV device plugin](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin) to be invoked when the
+> target pod is created.
+
+PR [kubevirt/kubevirt#13497](https://github.com/kubevirt/kubevirt/pull/13497) removed Namespace administrators'
+permissions to create VirtualMachineInstanceMigration objects - thus breaking the NIC hotplug / hotunplug feature
+for them in v1.5.
+
+## Motivation
+
+In previous KubeVirt versions, a namespace admin could hotplug and unplug NICs.
+We want to ensure that this feature will keep working in v1.5 and future versions and enhance it so the migration will be
+automatically performed - without user intervention.
+
+## Goals
+
+- Integrate the NIC hotplug / hotunplug with the LiveUpdate rollout strategy for consistent user experience.
+- Enable namespace admins to perform NIC hotplug / unplug operations independently - without involving the cluster-admin 
+for each hotplug / hotunplug operation.
+
+## Non Goals
+
+## Definition of Users
+
+- Namespace administrators
+
+## User Stories
+
+As a namespace admin I would like to be able to:
+- Hotplug / unplug a secondary interface without involving the cluster admin.
+- Hotplug a new secondary network interface that uses bridge binding.
+- Hotunplug a secondary network interface that uses bridge binding.
+- Hotplug a new secondary network interface of type SR-IOV.
+- Use the staging rollout strategy so any hotplug / hotunplug operation will take effect only after restarting a VM.
+
+## Repos
+
+- kubevirt/kubevirt
+
+## Design
+
+### Integrate with LiveUpdate Rollout Strategy
+
+The CPU and memory hotplug features use the `LiveUpdate` [rollout strategy](https://kubevirt.io/user-guide/user_workloads/vm_rollout_strategies/)
+which automatically initiates a migration of the VM when certain spec fields are changed.
+
+Extend the feature to automatically initiate a live migration in case of NIC hotplug / hotunplug request.
+
+#### API Additions
+
+Currently, the CPU, memory and volume hotplug / hotunplug flows have a dedicated condition on the VMI object to mark the VMI
+for automatic migration:
+- HotVCPUChange
+- HotMemoryChange
+- VolumesChange
+
+Since we do not wish to introduce another resource-specific condition, a new generic allowed value will be added to `VirtualMachineInstanceConditionType`:
```

Comment:
> Since we do not wish

I think this requires reasoning in order to convince reviewers.
You could point to an appendix in order to elaborate on this.